### DISCUSSION TITLE
Unreal Engine 5 support

### DISCUSSION THREAD

**Discussion by rawuser** [nxrighthere/UnrealCLR#267] [Help]
Hello,

did anyone tried to run UnrealCLR in Unreal 5? Currently I have Unreal 5.0.3 and I am trying to install and run plugin. However when I run project it throw messages that:
`%Project could not be compiled. Try rebuilding from source manually.`

Evene if I did and rebuild again solution from repository\Source\Managed\Runtime and copy it to project the issue persist.

Of course I tried Manual Compilation from tutorial. It didnt help.

Thanks

**Comment by nxrighthere** [comment]
Please, see https://github.com/nxrighthere/UnrealCLR/pull/248.
